schema/reportdb/update-reportdb-messages.txt

```diff
@@ -0,0 +1,6 @@
+
+SUSE Manager Report Database Schema Update
+
+This patch updates the report database schema.
+The patch will be applied automatically at the end
+of the update.
```

Comment:
I think we can drop this message.
We had this message for the schema at a time when we did not automate it to remind the admin that he had to apply it manually.
After we applied it automated, we just changed the message, but did not drop it yet.

We should not introduce it for the new package and we should think about dropping it also for the old one.